models/result_test.go

```diff
@@ -0,0 +1,195 @@
+package models_test
+
+import (
+	"go-google-scraper-challenge/initializers"
+	"go-google-scraper-challenge/models"
+	"go-google-scraper-challenge/models/results"
+	. "go-google-scraper-challenge/tests/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Result", func() {
+	Describe("CreateResult", func() {
```

Comment:
Should we have a test case to create result with specific status to make sure that status doesn't get override by the default one?

Reply:
@gutakk added in [416f1df](https://github.com/carryall/go-google-scraper-challenge/pull/42/commits/416f1dfd8f048d5955ab9691f6d0e0a2e3f2af7c)